### ISSUE TITLE
Suggestion, change "invis disabled", to have "break" available on silver edge.

### ISSUE THREAD

**Issue by Techmind** [darklordabc/Legends-of-Dota-Redux#1411]
Where is not enough items/spells (SD ulty + single item + viper skill) which can break passives (and they CAN BE banned too) so passive heavy tanks (timber passive + husk passive + dragon armor/dispersion/something else) are hard to disable from LONG range (600+ ), cause silver is not available, if invis is disabled (and in 90% of games it is) and castable passive disable is too short-ranged (vs typical blink or blink + stun is 1.2k range initiation). 
Also tanks are str heroes usually so they have GREAT status resistance too. So you might not have ebought time to kill them after hexing/stunning.

So change 'invis' disabled to HAVE silber edge-2 which gives same benefits as usual silver edge, but applies different effect which does not makes hero invis BUT have same passive disabled on hit.

**Comment by darklordabc** [comment]
You know there is a silver item? A new item added for Redux that can break passives right?

**Comment by Techmind** [comment]
I know about the item, buts its range is like 400 or something like this ? On anything which takes cast range talent or spell amp talent, you DONT want to get in melee range to disable huskars passive or dispersion (because mages are too squishy to survive 1-3 right clicks).

**Comment by Techmind** [comment]
Also silver`s items stats dont give % ms on cast and overall have way-worse stats so very slot-inefficient for +10 stats.

**Comment by Techmind** [comment]
Made a picture - https://imgur.com/a/ExpOS. Basicly cast range is too short (even with cast range talent + aethers), you are required to get into range of 500-700 units  of melee hero (right clicker) on any caster to use it.

This is 2 seconds of walking with 550 ms for ANY agi hero and for any str hero your stun(assuming y have one ) will work for 0.5-1.5 seconds. So unless you can insta-wipe them and that hero is alone, you are instanly dead. 

The only counters to passive/tanky builds is 
1) CAST range talent + SD ULT/viper skill (assuming they are itemless), 
2) attack range talent + silver edge. 
3) Being more tanky/more dps.
4) Being able to do STUPid amounts of pure dmg and having LOTS of str to ignore blademail-> od orb OR ench ult.
5) Instakill spells basically linken popper + decrepify + ancient seal + huskars ult + passive spell amp(so at least int hero can somewhat scale late game) + ethereal/veil/orchid to counter 1-2 pipes + some way to counter blademail/dispersion. <- And y cant push  with this build anyway so loose the game.

TLDR: need passive disabled with range comprarable to blink at least. So y can blink in-non melee range + unload spells + run away.

**Comment by darklordabc** [comment]
Heres what I'll do.
*Decrease the gold cost of silver and silver consumable by 250 gold.
*Cast range increased to 800 from 600
*Damage reduction increased from 25% to 35%

What you think of that?

**Comment by Techmind** [comment]
Looks better, but usual melee's best mobility/stun skill is sk`s stun + possibly aghs OR no aghs but possible blink. So initiation range is 800/800+1250/1450 minimum for melees.

At least aethers + 800 range sort of better than 1st, maybe we should add night vision (+200, i think avg is 600-800) bonus so at night you can cast it from max range for example ?

Also proper treatment can be to decreasing cast points (for all apells), cause melee heroes have almost instant attack late game anyway. 
But mages still have to cast spells with projectile times/cast points/etc.
